src/index.ts

```diff
@@ -10,18 +11,25 @@ const dotsimus = new Client({
   ],
 });
 
-if (!process.env.DISCORD_TOKEN) {
-  throw new Error("Discord token not provided.");
+
+if (!process.env.NODE_ENV) {
+  process.env.NODE_ENV = 'production';
```

Comment:
Please don't default to production for NODE_ENV. Either throw an error, or (if you must) default to development.